src/reporters/base.js

```diff
@@ -120,8 +121,10 @@ export default class BaseReporter {
             testRun.errs.forEach(err => err.userAgent = userAgent);
 
             reportItem.pendingRuns--;
-            reportItem.errs     = reportItem.errs.concat(testRun.errs);
-            reportItem.unstable = reportItem.unstable || testRun.unstable;
+            reportItem.errs           = reportItem.errs.concat(testRun.errs);
+            reportItem.unstable       = reportItem.unstable || testRun.unstable;
+            reportItem.hasScreenshots = reportItem.hasScreenshots || testRun.hasScreenshots;
+            reportItem.screenshotPath = reportItem.screenshotPath || testRun.test.screenshotPath;
```

Comment:
It will be semantically correct to assign `screenshotPath` once in the `_createReportItem`